android/app/src/main/res/values-es/strings.xml

```diff
@@ -2237,4 +2220,61 @@
+    <string name="telegram">Telegram</string>
+    <string name="twitter">X (Twitter)</string>
+    <string name="instagram">Instagram</string>
+    <string name="wikimedia_commons">Wikimedia Commons</string>
+    <string name="telegram_url">https://t.me/OrganicMapsApp</string>
+    <string name="instagram_url">https://www.instagram.com/organicmaps.app</string>
+    <string name="type.highway.world_towns_level">highway-world_towns_level</string>
+    <string name="type.hwtag">hwtag</string>
+    <string name="type.hwtag.nofoot">hwtag-nofoot</string>
+    <string name="type.hwtag.private">hwtag-private</string>
+    <string name="type.hwtag.toll">hwtag-toll</string>
+    <string name="type.hwtag.yesbicycle">hwtag-yesbicycle</string>
+    <string name="type.psurface.paved_good">psurface-paved_good</string>
+    <string name="type.psurface.unpaved_good">psurface-unpaved_good</string>
+    <string name="type.railway.abandoned.bridge">Abandoned Railway Bridge</string>
+    <string name="type.railway.funicular.tunnel">Funicular Tunnel</string>
+    <string name="type.railway.funicular.bridge">Funicular Bridge</string>
+    <string name="type.railway.monorail.tunnel">Túnel del Monorraíl</string>
+    <string name="type.railway.narrow_gauge.bridge">Narrow Gauge Rail Bridge</string>
+    <string name="type.railway.narrow_gauge.tunnel">Narrow Gauge Rail Tunnel</string>
+    <string name="type.railway.preserved.bridge">Preserved Rail Bridge</string>
+    <string name="type.route.shuttle_train">route-shuttle_train</string>
+    <string name="type.railway.preserved.tunnel">Preserved Rail Tunnel</string>
+    <string name="type.hwtag.oneway">hwtag-oneway</string>
+    <string name="type.hwtag.nobicycle">hwtag-nobicycle</string>
+    <string name="facebook">Facebook</string>
+    <string name="type.hwtag.lit">hwtag-lit</string>
+    <string name="type.psurface.unpaved_bad">psurface-unpaved_bad</string>
+    <string name="comma_separated_pair">%1$s, %2$s</string>
+    <string name="openstreetmap">OpenStreetMap</string>
+    <string name="type.mapswithme">mapswithme</string>
+    <string name="type.mapswithme.grid">mapswithme-grid</string>
+    <string name="type.hwtag.yescar">hwtag-yescar</string>
+    <string name="type.railway.preserved">Preserved Rail</string>
+    <string name="matrix">[Matrix]</string>
+    <string name="type.man_made.petroleum_well">Oil or gas well</string>
+    <string name="type.man_made.flare">Gas Flare</string>
+    <string name="type.psurface">psurface</string>
+    <string name="type.railway.abandoned.tunnel">Abandoned Railway Tunnel</string>
+    <string name="type.railway.light_rail.bridge">Light Rail Bridge</string>
+    <string name="type.railway.light_rail.tunnel">Light Rail Tunnel</string>
+    <string name="editor_line_social_network">LINE</string>
+    <string name="type.emergency.lifeguard">Lifeguard</string>
+    <string name="type.emergency.mountain_rescue">Mountain Rescue Station</string>
```

Comment:
This is English not Spanish, are the strings coming in from a presumptive english default?

Reply:
These particular new POIs are not translated yet https://github.com/organicmaps/organicmaps/pull/10286